tutorials/tour/basics.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Running Scala on Browser
```

Comment:
Maybe "Trying Scala in the Browser" to be clear it's not a feature of the language but rather a way to try it out.